jib-core/src/main/java/com/google/cloud/tools/jib/docker/DockerContextGenerator.java

```diff
@@ -60,31 +60,54 @@ public DockerContextGenerator(SourceFilesConfiguration sourceFilesConfiguration)
   /**
    * Sets the base image for the {@code FROM} directive. This must be called before {@link
    * #generate}.
+   *
+   * @param baseImage the base image.
+   * @return The {@link DockerContextGenerator} with the base image set.
```

Comment:
nit: lowercase for start of `@...` docs and no period

Reply:
Oh, should we not be using periods? It seems to be 50/50 everywhere so I've been adding them.

Reply:
Yeah, I believe the style is to only use a period for these block tags if you have a second sentence afterwards. I can't quite find where this convention was stated at though...